docs/technical_reference/unit_models/electrocoagulation.rst

```diff
@@ -0,0 +1,371 @@
+.. _EC_0D:
+
+Electrocoagulation (0D)
```

Comment:
Should filename across the board be `electrocoagulation_0D`, aligning with naming for other units?

Reply:
I thought about this and I am happy to make the change. Though after surveying other unit model names, I don't really think this is an established convention in our unit model naming to a significant extent (e.g.,  gac, electrolyzer, clarifier, etc.).

For that matter, maybe I should remove the `(0D)` from these docs title?